sdk/nodejs/apigateway/vpcLink.ts

```diff
@@ -75,7 +75,7 @@ export interface VpcLinkState {
     /**
      * The list of network load balancer arns in the VPC targeted by the VPC link. Currently AWS only supports 1 target.
      */
-    readonly targetArn?: pulumi.Input<string>;
+    readonly targetArn?: pulumi.Input<pulumi.Input<string>>;
```

Comment:
@lukehoban this doesn't look like a good change.  Did something change in our tf layer that is causing this?

Reply:
I think this is: https://github.com/pulumi/pulumi-terraform/pull/217